src/WebJobs.Script.WebHost/Security/HostKeyScopes.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost
+{
+    public static class HostKeyScopes
```

Comment:
This was kept as string constants because the API expects the scope to be a string (at the function level, the function name is the scope). We could `ToString` when calling the API, but the enum value here is reduced.